ansible_builder/constants.py

```diff
@@ -14,8 +14,8 @@
 
 build_arg_defaults = dict(
     ANSIBLE_GALAXY_CLI_COLLECTION_OPTS='',
-    EE_BASE_IMAGE='quay.io/ansible/ansible-runner:devel',
-    EE_BUILDER_IMAGE='quay.io/ansible/ansible-builder:latest'
+    ANSIBLE_RUNNER_IMAGE='quay.io/ansible/ansible-runner:devel',
```

Comment:
Raising this point out of a sense of obligation:

https://github.com/ansible/ansible-builder/issues/213

If we, in the future, allow building EEs with other EEs as the base image (as opposed to the ansible-runner image), then this naming will age poorly.

How interested are we in that feature? I'm not sure. If we don't intend to support it, then your naming here would help keep people from trying it.